components/tag/chip/src/index.scss

```diff
@@ -0,0 +1,21 @@
+
+@import '~@schibstedspain/theme-basic/lib/index';
+
+.sui-TagChip {
+  @include sui-tag;
+
+  &-link {
+    @include reset-link;
+    cursor: pointer;    
+  }
+  &-delete-icon {
+    @include reset-button;
+    height: 15px;
+    margin-bottom: -3px;
```

Comment:
El otro día pasé en el canal de frontend de Slack un artículo sobre cómo alinear iconos SVG y texto: https://blog.prototypr.io/align-svg-icons-to-text-and-say-goodbye-to-font-icons-d44b3d7b26b4#.d1weah9lw. Evitaría hacer este tipo de alineación con valores absolutos, no es muy flexible.